apps/dashboard/src/app/App.tsx

```diff
@@ -52,11 +52,16 @@ export class App extends React.Component<IAppSetting, IAppState> {
         >
           {this.state.application === "interpret" && (
             <Interpret
-              dataset={
+              data={
                 applications[this.state.application].datasets[
                   this.state.dataset
                 ].data
               }
+              dataset={
```

Comment:
At the time of comment, the `IDataset` interface looks like this:
![image](https://user-images.githubusercontent.com/64443771/148107497-77bf3aa3-be93-449a-a0ef-dad4a6ee2ce7.png)